README.md

```diff
@@ -1,2 +1,145 @@
-# Leaderboard-setup-project
-Leaderboard: setup project
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:movie_camera: Project Presentation](#project-presentation)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+
+**[Leaderboard ]** "Leaderboard" is a JavaScript project focused on creating a list app using webpack and ES6 features, particularly modules. The initial version aims to implement core functionality based on a wireframe, prioritizing functionality over styling. Subsequent activities will involve integrating the Leaderboard API using JavaScript async and await, and enhancing the app with styling elements.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS and Javascript vanilla.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Plain JS with objects]**
+- **[JavaScript classes]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presntation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+Walking through portfolio outline.
+
+- [Project Presentation Link] [#](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jlberbesi.github.io/Awesome_books.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Please, kindly update the live demo link by removing it as it links to a different project which doesn't make your readme professional.

Reply:
@zieeco My bad.  Thanks for letting me know.  Wondering if we need something  on the live version at this point.